src/main/resources/static/templates/XSSWithHtmlTagInjection/LEVEL_1/XSS.js

```diff
@@ -1,21 +1,20 @@
-function addingEventListenerToLoadImageButton() {
-  document.getElementById("submit").addEventListener("click", function () {
-    let url = getUrlForVulnerabilityLevel();
-    doGetAjaxCall(
-      appendResponseCallback,
-      url + "?value=" + document.getElementById("textInput").value,
-      false
-    );
-  });
-}
-addingEventListenerToLoadImageButton();
-
 function appendResponseCallback(data) {
   var parentContainer = document.getElementById("parentContainer");
-  parentContainer.innerHTML = data;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities